setup.py

```diff
@@ -56,7 +56,8 @@
         "grpcio >= 1.38.1, < 2.0dev",
         "pyarrow >= 1.0.0, < 6.0dev",
     ],
-    "pandas": ["pandas>=0.23.0", "pyarrow >= 1.0.0, < 6.0dev"],
+    "pandas": ["pandas>=0.24.2", "pyarrow >= 1.0.0, < 6.0dev"],
+    "geopandas": ["geopandas>=0.9.0, <1.0dev", "shapely>=1.6.0, <2.0dev"],
```

Comment:
This seems a rather narrow range. Would `0.7.0` or `0.8.0` which were both released in 2020 work as a minimum for `geopandas`?

Shapely seems plenty wide, since `1.6.0` was released in 2017, but should you use a capital `S` to reflect the package name here? https://pypi.org/project/Shapely/1.6.0/

Reply:
I tested the ranges. <0.9.0 didn't work (due to changes in how CRS was tracked).

Package lookup is case insensitive, but I capitalized Shapely.